style.css

```diff
@@ -0,0 +1,869 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lato:ital,wght@0,300;0,700;1,700&display=swap"');
+
+* {
+  margin: 0;
+  padding: 0;
+  transition: all ease 0.15s;
+  font-family: 'Lato', sans-serif;
+}
+
+nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: #fff;
+  z-index: 999;
+}
+
+.ham,
+.closes {
+  display: none;
+}
+
+.nav-menu {
+  display: none;
+}
+
+.d-flex {
+  display: flex;
+  align-items: center;
+}
+
+.upper-nav {
+  background-color: #272a31;
+  padding: 5px 0;
+}
+
+.partners img {
+  width: 110px;
+  padding: 20px 0 20px 0;
+}
+
+.logo-container img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+  margin-bottom: 20px;
+}
+
+.iconCan {
+  width: 300px;
+  height: 100%;
+  margin: 0 20px 0 auto;
+  gap: 20px;
+}
+
+.iconCan :nth-child(1) {
+  width: 9px;
+}
+
+.subLeft img {
+  width: 160px;
+  height: 60px;
+  object-fit: cover;
+  object-position: center;
+}
+
+.end-logo img {
+  position: relative;
+  width: 100px;
+  padding: 0 0 0 120px;
+  object-fit: cover;
+}
+
+.iconCan :nth-child(2) img {
+  width: 16px;
+}
+
+.iconCan a {
+  color: #d3d3d3;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 550;
+}
+
+.main-nav {
+  /* background-color: red; */
+  box-shadow: 0 2px 5px 2px rgba(160, 160, 160, 0.5);
+  padding: 8px 0;
+}
+
+.subNav {
+  width: 70%;
+  margin: 0 auto;
+
+  /* background-color: blue; */
```

Comment:
[OPTIONAL] Leaving unused code as comments in your codebase is generally not recommended. It can lead to confusion, increase code clutter, and make code maintenance more difficult. Kindly remove all commented code from your codebase to make it look more professional. :+1: